src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-form/osd-form.component.html

```diff
@@ -1,136 +1,212 @@
 <cd-orchestrator-doc-panel *ngIf="!hasOrchestrator"></cd-orchestrator-doc-panel>
 
-<div class="cd-col-form"
+<div class="card"
      *cdFormLoading="loading">
-  <form name="form"
-        #formDir="ngForm"
-        [formGroup]="form"
-        novalidate>
-    <div class="card">
-      <div i18n="form title|Example: Create Pool@@formTitle"
-           class="card-header"
-           *ngIf="!hideTitle">{{ action | titlecase }} {{ resource | upperFirst }}</div>
-      <div class="card-body">
-        <fieldset>
-          <cd-osd-devices-selection-groups #dataDeviceSelectionGroups
-                                           name="Primary"
-                                           type="data"
-                                           [availDevices]="availDevices"
-                                           [canSelect]="availDevices.length !== 0"
-                                           (selected)="onDevicesSelected($event)"
-                                           (cleared)="onDevicesCleared($event)">
-          </cd-osd-devices-selection-groups>
-        </fieldset>
+  <div i18n="form title|Example: Create Pool@@formTitle"
+       class="card-header"
+       *ngIf="!hideTitle">{{ action | titlecase }} {{ resource | upperFirst }}</div>
+  <div class="card-body ml-5">
+    <div class="accordion">
+      <div class="card">
+        <div class="card-header">
+          <h2 class="mb-0">
+            <button class="btn btn-link btn-block text-left dropdown-toggle"
+                    data-toggle="collapse"
+                    aria-label="toggle deployment options"
+                    [attr.aria-expanded]="accordionShow"
+                    (click)="accordionShow = !accordionShow">Deployment Options</button>
+          </h2>
+        </div>
+      </div>
+      <div class="collapse"
+           [ngClass]="{show: accordionShow}">
+        <div class="card-body flex-this">
+          <div class="size-card">
+            <div class="form-check">
+              <input class="form-check-input"
+                     type="radio"
+                     name="sh"
+                     id="flexRadioDefault1">
+              <label class="form-check-label"
+                     for="flexRadioDefault1">Cost/Capacity-optimized (Recommened)</label>
```

Comment:
@nizamial09 why **Cost/Capacity-optimized** is Recommended by default?
For the given three options recommened should come by based on the number of hosts and type of devices user is going to deploy the OSD's Is it?

Reply:
@sunilangadi2 this PR is still in WIP and the backend integration isn't done yet. Its being worked by @pereman2 and when he finishes it we'll start integrating it together. I meant this PR as a skeleton for the coming works. 



> For the given three options recommened should come by based on the number of hosts and type of devices user is going to deploy the OSD's Is it?

Yup, it'll be.